tests/test.h

```diff
@@ -285,8 +292,24 @@ class BloatyTest : public ::testing::Test {
     return nullptr;
   }
 
+
+  const std::string ObjExtension() const {
+    if (pe_test_) {
+      return "obj";
+    }
+    return "o";
+  }
+
+  const std::string SharedObjectExtension() const {
+    if (pe_test_) {
+      return "dll";
+    }
+    return "so";
+  }
```

Comment:
For the ones where linux used native extensions, I used windows extensions.
I also used .bin on the windows executable, since a lot of windows git configurations explicitly exclude *.exe